packages/jest-expo/src/preset/expoModules.js

```diff
@@ -772,14 +774,8 @@ module.exports = {
         },
         ExponentFileSystem: {
           addListener: { type: 'function' },
-          bundleDirectory: {
-            type: 'object',
-            mock: null,
-          },
-          bundledAssets: {
-            type: 'object',
-            mock: null,
-          },
+          bundleDirectory: { type: 'object', mock: null },
```

Comment:
Interesting that this changed. Why would you and the person who last formatted this file have different Prettier settings 🤔